corehq/apps/sso/views/accounting_admin.py

```diff
@@ -0,0 +1,78 @@
+from corehq.apps.accounting.dispatcher import AccountingAdminInterfaceDispatcher
+from corehq.apps.accounting.filters import (
+    DateCreatedFilter,
+    NameFilter,
+)
+from corehq.apps.accounting.interface import AddItemInterface
+from corehq.apps.reports.datatables import DataTablesHeader, DataTablesColumn
+from corehq.toggles import ENTERPRISE_SSO
+
+from corehq.apps.sso.models import IdentityProvider
+
+
+class IdentityProviderInterface(AddItemInterface):
+    section_name = 'Accounting'
+    name = 'Identity Providers (SSO)'
+    description = 'For managing identity providers for Single Sign On (SSO)'
+    slug = 'identity_providers'
+    dispatcher = AccountingAdminInterfaceDispatcher
+    hide_filters = False
+    item_name = "Identity Provider"
+
+    fields = [
+        'corehq.apps.accounting.interface.DateCreatedFilter',
+        'corehq.apps.accounting.interface.NameFilter',
+    ]
+
+    @classmethod
+    def show_in_navigation(cls, domain=None, project=None, user=None):
+        return ENTERPRISE_SSO.enabled(user.username)
```

Comment:
this it's part of the base class method's sig. some reports are used anonymously in places I guess? I'm not quite sure why `user=None`. Maybe it's for when only `domain` or `project` are available (which would never be the case for an accounting admin UI)

Reply:
If `user=None` is really just a vestige of old code, and we're confident that the user can never actually be None, can we leave a comment indicating that it can't really be None? Otherwise I think it that case needs to be handled in some way.

Reply:
it can be `None` though if `domain` or `project` are filled in. it depends on the report, and it's been so mangled I'm not 100% sure what the current state is and no there aren't enough tests to reliably make the change LOL 😭